docs/samples/extension/config/tekton-extension-sample-service.yaml

```diff
@@ -0,0 +1,19 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: dashboard-extension
+  labels:
+    tekton-dashboard-extension: "true"
+  annotations:
+    tekton-dashboard-endpoints: "/extensions/sample"
+    tekton-dashboard-display-name: "tekton_dashboard_extension"
+    tekton-dashboard-bundle-location: "/bundle"
```

Comment:
I won't be able to test this until Monday at the earliest, but https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/ states, 
> The name segment is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between. 

Are the / characters valid and accepted - do they work?

Reply:
Yes, it works.